rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec

```diff
@@ -38,8 +38,8 @@ pushd src
 make clean
 
 # Build Wazuh sources
-make deps PREFIX=%{_localstatedir} TARGET=server
-make -j%{_threads} TARGET=server USE_SELINUX=yes USE_FRAMEWORK_LIB=yes PREFIX=%{_localstatedir} DEBUG=%{_debugenabled}
+make deps TARGET=server
+make -j%{_threads} TARGET=server USE_SELINUX=yes DEBUG=%{_debugenabled}
```

Comment:
Is `USE_FAMEWORK_LIB` also out?

Reply:
As indicated in the issue #611 `USE_FAMEWORK_LIB` is deprecated.